packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -219,6 +219,16 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   /* @conditional-compile-remove(file-sharing-acs) */
   const attachmentUploadButtonOnChange = useCallback(
     (files: FileList | null): void => {
+      // look up sendbox by ID for now, we will use `useRef`
+      // once attachment button is moved inside of send box component
+      let sendBox = document?.querySelector(`[id="sendbox"]`) as HTMLTextAreaElement;
+      /* @conditional-compile-remove(rich-text-editor-composite-support) */
+      if (props.options?.richTextEditor) {
+        sendBox = document?.querySelector(`[id="richTextSendBox"]`) as HTMLTextAreaElement;
```

Comment:
rich text send box is a `div` with `contenteditable="true"`, not `HTMLTextAreaElement`

Reply:
thanks fixed